services/scripts.py

```diff
@@ -65,3 +70,9 @@ def get_minimum_and_maximum(data: list[list[str]], index: int) -> tuple[float, f
 
 def get_shape(data: list[list[str]]) -> tuple[int, int]:
     return len(data), len(data[0])
+
+
+if __name__ == '__main__':
+    db = DataBase(filename='../data/Spotify_Youtube.csv')
+
+    print(db.get_song(1).name)
```

Comment:
What if get_song returns None? This will cause an error: NoneType obj has no attr name

Reply:
So... I wrote that func can return none. I can write return of an empty object, but why?

Reply:
The problem is not in type hints. get method can return obj or None. At line 78 you say, form get_song take attr name. The problem is, that get_song can be both song and None. Does None has an name attr?

Reply:
It's just example.. I can remove it or fix it..